! LaTeX Error: File `IEEEtran.cls' not found.

```
! LaTeX Error: File `IEEEtran.cls' not found.

Type X to quit or <RETURN> to proceed,
or enter new name. (Default extension: cls)

Enter file name: 
! Emergency stop.
<read *> 
         
l.2 \usepackage

pandoc: Error producing PDF
makefile:16: recipe for target 'pdf' failed
make: *** [pdf] Error 43
```

--- a/README.md
+++ b/README.md
@@ -11,7 +11,7 @@
 
 ```sh
 sudo apt update
-sudo apt install pandoc pandoc-citeproc texlive-full
+sudo apt install pandoc pandoc-citeproc texlive-full texlive-publishers
 ```
 
 ### Fedora

